platform/android/MapboxGLAndroidSDK/src/main/res/values/strings.xml

```diff
@@ -11,11 +11,14 @@
     <string name="infoWindowAddress">Address</string>
 
     <!-- these are public -->
-    <!-- {@deprecated Use Style.getXStyleUrl(int version) instead.} -->
+    <!-- Using one of these constants means your map style will always use the latest version and
+     may change as we improve the style. -->
```

Comment:
Same here about being super clear that this means Style will change.  Maybe we should put some language in both the Java and XML side about these being "Convenience" Style URLs, and that the preferred route is to use regular Style URLs